docs/tutorials/blog.md

```diff
@@ -19,9 +19,9 @@ Click this button to create a [Gitpod][gitpod] workspace with the project set up
 
 If you want to follow this tutorial locally on your own computer, it is important for you to have these things installed:
 
-- [Node.js][node-js] version (^12.22.0, ^14.17.0, or >=16.0.0)
+- [Node.js][node-js] version (^14.17.0, or >=16.0.0)
 - [npm][npm] 7 or greater
-- A code editor
+- A code editor ([VSCode][vs-code] is a nice one)
```

Comment:
WebStorm also, but I'm not sure this belongs in this PR 🙈

Reply:
I just copied this over from the jokes tutorial since I saw a slight difference here